recipe/meta.yaml

```diff
@@ -23,11 +23,14 @@ requirements:
     - pip
   run:
     - python
+    - setuptools >=36
 
 test:
   imports:
     - markdown
     - markdown.extensions
+    - coverage
+    - pyyaml
```

Comment:
Why coverage and pyyaml here?

Reply:
these packages are listed under testing in extras_require in setup.py. Should I omit these instead?

Reply:
Well, those would be extra requirements if you would run the unittests here ... but `imports` have nothing to do with running unittests. Thus they can be removed here.

Reply:
See also:
* https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-requirements 
* https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands 

But that is not required here.

Reply:
The import tests are just doing something like `python -c "import markdown, markdown.extensions"`... which is normally enough.